Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1307,6 +1314,25 @@ class BrowserViewController: UIViewController {
     }
 }
 
+extension BrowserViewController: ClipboardBarDisplayHandlerDelegate {
+    func shouldDisplayClipboardBar(absoluteString: String) {
+        clipboardToast = ButtonToast(labelText: Strings.GoToCopiedLink, descriptionText: absoluteString,  buttonText: Strings.GoButtonTittle, completion: { (buttonPressed) in
```

Comment:
Alright last one! if you make BVC delegate conform to the `SettingsDelegate` (which it already does. I know its a bad name) then you can create this toast in the `ClipboardBarDisplayHandler.swift` and avoid having to have this code here. Then shouldDisplayClipboardBar would just return a toast and all BVC would have to do is display it :)